src/containers/Swatch/Swatch.js

```diff
@@ -19,8 +19,8 @@ class Swatch extends Component {
 		const paletteDisplay = palette.length && 
 			palette.map((element, index) => {
 				return <div className='color' key={index} data-key={index} style={{backgroundColor: element.color}}>
-				<h1>{element.color}</h1>
-				<h2 onClick={this.handleLock}>{element.locked ? 'true' : 'false'}</h2>
+				<h1 className='hex-code'>{element.color}</h1>
+				<h2 onClick={this.handleLock}>{element.locked ? 'release' : 'catch'}</h2>
```

Comment:
I like the naming conventions of 'release' and 'catch' - it goes well with the name of our project.